apis/v1alpha1/gatewayclass_types.go

```diff
@@ -33,15 +33,19 @@ import (
 // for creating Gateway resources.
 //
 // GatewayClass is a Cluster level resource.
-//
-// Support: Core.
```

Comment:
Support level is removed since no other resources specify it. Fields should indicate the support-level and not the top-level resource.

Reply:
I think there may be a place for this, especially for route types, but I'm not exactly sure what that should be. I think we want a world where both L4 and L7 implementations can support different parts of this API. I don't mind removing this here though since it doesn't seem like we've added it for other resources.